src/tests/ftest/pool/RebuildTests.py

```diff
@@ -236,118 +201,69 @@ def test_multipool_rebuild(self):
-            createsetid = self.params.get("setname",
-                                          '/run/testparams/createset/')
-            createsize = self.params.get("size", '/run/testparams/createsize/')
-
             # initialize python pool object then create the underlying
             # daos storage, the way the code is now the pools should be
             # on the same storage and have the same service leader
-            pool1 = DaosPool(self.Context)
-            pool2 = DaosPool(self.Context)
-            pool1.create(createmode, createuid, creategid,
-                         createsize, createsetid, None)
-            pool2.create(createmode, createuid, creategid,
-                         createsize, createsetid, None)
+            pool1 = DaosPool(self.context)
+            pool2 = DaosPool(self.context)
+            pool1.create(self.createmode, self.createuid, self.creategid, self.createsize,
+                         self.createsetid)
+            pool2.create(self.createmode, self.createuid, self.creategid, self.createsize,
+                         self.createsetid)
 
             # want an open connection during rebuild
             pool1.connect(1 << 1)
             pool2.connect(1 << 1)
 
             # create containers
-            container1 = DaosContainer(self.Context)
+            container1 = DaosContainer(self.context)
             container1.create(pool1.handle)
-            container2 = DaosContainer(self.Context)
+            container2 = DaosContainer(self.context)
             container2.create(pool2.handle)
 
             # now open them
             container1.open()
             container2.open()
 
-            # how many objects and records are we creating
-            objcount = self.params.get("objcount",
-                                       '/run/testparams/numobjects/*')
-            reccount = self.params.get("reccount",
-                                       '/run/testparams/numrecords/*')
-            if objcount == 0:
-                reccount = 0
-
-            # which rank to write to and kill
-            rank = self.params.get("rank", '/run/testparams/ranks/*')
-
-            # how much data to write with each key
-            size = self.params.get("size", '/run/testparams/datasize/')
-
             # Putting the same data in both pools, at least for now to simplify
             # checking its correct
             saved_data = []
-            for i in range(0, objcount):
+            for _objc in range(self.objcount):
                 obj = None
-                for j in range(0, reccount):
+                for _recc in range(self.reccount):
 
                     # make some stuff up and write
-                    dkey = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
+                    dkey = ''.join(random.choice(string.ascii_uppercase + string.digits)
```

Comment:
(pylint-line-too-long) Line too long (88/80)

Reply:
no belong to current patch.